app/src/main/java/com/example/cinemapp/ui/main/ProfileViewModel.kt

```diff
@@ -0,0 +1,8 @@
+package com.example.cinemapp.ui.main
+
+import androidx.lifecycle.ViewModel
+
+class ProfileViewModel : ViewModel() {
```

Comment:
Looks empty 😅

Reply:
Moved some things there as per some of the previous comments, the rest will be added in a following ticket.